pywbem_mock/_dmtf_schema.py

```diff
@@ -0,0 +1,392 @@
+#
```

Comment:
The new module is not showing up in the documentation.

Reply:
Added that and also moved documentation around to avoid duplications

Reply:
DONE